### PR TITLE
Add remote_branch option and increase the childProcess.execSync buffer size

### PR DESCRIPTION
**Pull request by Dhina17**
- Needed.

### CODE REVIEW

**Comment by Dhina17** [comment]
issue #52

File: README.md [hunk 1]
```diff
@@ -240,6 +241,35 @@ jobs:
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: akhileshns/heroku-deploy@v3.8.9 # This is the action
+        with:
+          heroku_api_key: ${{secrets.HEROKU_API_KEY}}
+          heroku_app_name: "YOUR APP's NAME" #Must be unique in Heroku
+          heroku_email: "YOUR EMAIL"
+          branch: "main"
+```
+
```
Comment (AkhileshNS) [inline README.md:270]: Hey could you provide an example using the `remote_branch` property for reference. The current one seems to be using `branch` which I think is a typo

**Comment by AkhileshNS** [comment]
Also @Dhina17, I will need to merge an earlier pull request, so you may have to pull some changes and update your branch. Thanks again for your interest in the action

File: README.md (hunk 1 above)
Comment (Dhina17) [reply README.md:270]: sure.

**Comment by Dhina17** [comment]
> Also @Dhina17, I will need to merge an earlier pull request, so you may have to pull some changes and update your branch. Thanks again for your interest in the action

Ok I will do after your merge

**Comment by Dhina17** [comment]
@AkhileshNS Done

**Comment by leonid-shevtsov** [comment]
Hey I don't think the buffer size was changed at all. Which is a pity, because we're still getting ENOBUFS.

**Pull request merged**